addon/mixins/route.js

```diff
@@ -168,11 +168,13 @@ export default Ember.Mixin.create({
         this.set('_currentPage', startingPage);
         this.set('_totalPages', totalPages);
         infinityModel.set('reachedInfinity', !this.get('_canLoadMore'));
-        Ember.run.scheduleOnce('afterRender', this, 'infinityModelUpdated', {
-          lastPageLoaded: startingPage,
-          totalPages: totalPages,
-          newObjects: infinityModel
-        });
+        if(this.infinityModelUpdated) {
```

Comment:
Ahh right.  So the dev has to define these functions on the route for the callback to actually run.  I like it!

@kellyselden / @mike-north can u guys confirm that sounds correct?